components/molecules/GamePlayedStory/GamePlayedStory.tsx

```diff
@@ -1,61 +1,91 @@
 import React from 'react'
-import {Pressable, StyleSheet, Text, View, ViewStyle} from "react-native"
+import {Pressable, ScrollView, StyleSheet, View} from "react-native"
 import GameCard from '../GameCard/GameCard'
-import {FriendCircle} from '../Friend/FriendGroup'
-import {FriendNetworkStatus} from '../Friend/Friend'
+import FriendGroup, {FriendCircle, FriendGroupItem} from '../Friend/FriendGroup'
+import Title from '../../atoms/Title/Title'
 
 export interface GamePlayedStoryProps {
-    title: string
-    style?: ViewStyle
-    gameBookImageUrl: string | undefined
-    gameBookTitle: string
-    onBookTapped: () => void
-    onStoryTapped: () => void
-    avatarImage: string | undefined
-    avatarUername: string
-    avatarStatus: FriendNetworkStatus
+    friends: FriendGroupItem[]
+    games: {imageUrl: string; name:string, onThumbnailTapped?: ()=> void}[]
+    timeString: string 
+    maxWidth?: number
 }
 
+const renderNames = (names: string[]) => 
+names.length === 2 ? names.join(" & ") : 
+names.length === 3 ? names.slice(0,2).join(", ") + " & " + names[2]:
+names.length > 3 ? names.slice(0,2).join(", ") + " & " + names[2] + " +" + (names.length -3) : names
+
 export const GamePlayedStory = (props: GamePlayedStoryProps) => {
+    const {
+        friends,
+        games,
+        maxWidth,
+        timeString
+    } = props
```

Comment:
As a stylistic convention, let's avoid this definition. Let's just refer to all of them below as `props.friends` and `props.games`. It's slightly more verbose but I think it's a helpful convention so that you always know when you are referencing a prop. Plus, it's how we've been doing it in our main codebase.